hircluster.c

```diff
@@ -100,7 +100,7 @@ static void freeRedisClusterNode(redisClusterNode *node);
 static void cluster_slot_destroy(cluster_slot *slot);
 static void cluster_open_slot_destroy(copen_slot *oslot);
 static int updateNodesAndSlotmap(redisClusterContext *cc, dict *nodes);
-static int updateSlotMapAsync(redisClusterAsyncContext *acc,
+static int updateSlotmapAsync(redisClusterAsyncContext *acc,
```

Comment:
Not sure if it's worth polluting the git history with this change (and the calls to it). It's an internal function so it's not visible in the public API anyway. In Redis they always reject such changes. WDYT?

Reply:
Sure, it would be nice to cleanup stuff when seen, but lets have smaller deliverables.